base/progs/gametypes/generic/matchstates.as

```diff
@@ -223,18 +175,3 @@ Entity @RandomEntity( String &className )
 		return null;
 	return spawnents[ random_uniform( 0, spawnents.size() ) ];
 }
-
-void GENERIC_UpdateMatchScore()
-{
-	if( gametype.isTeamBased ) {
```

Comment:
undecided between keeping this or letting the engine do it

for now let's keep it